src/mbgl/map/map.cpp

```diff
@@ -250,10 +250,20 @@ LatLngBounds Map::latLngBoundsForCamera(const CameraOptions& camera) const {
     Size size = shallow.getState().getSize();
 
     shallow.jumpTo(camera);
-    return LatLngBounds::hull(
-        shallow.screenCoordinateToLatLng({}),
-        shallow.screenCoordinateToLatLng({ double(size.width), double(size.height) })
-    );
+    LatLng nw = shallow.screenCoordinateToLatLng({});
```

Comment:
This change breaks API, it would be safer to introduce new method with descriptive name, e.g., `latLngBoundsForCameraUnwrapped`.

Reply:
@alexshalamov Thanks for the suggestion, I have moved the API breaking changes to a new method `latLngBoundsForCameraUnwrapped`.